bin/packages/build.js

```diff
@@ -116,7 +116,9 @@ function createStyleEntryTransform() {
 
 			packages.add( packageName );
 			const entries = await glob(
-				path.resolve( PACKAGES_DIR, packageName, 'src/*.scss' )
+				path
+					.resolve( PACKAGES_DIR, packageName, 'src/*.scss' )
+					.replace( /\\/g, '/' )
 			);
```

Comment:
I discovered that when I run npm run dev, `createStyleEntryTransform` function is executed and the CSS file stored in this `entries` variable is returned to stream.
However, this variable was empty on Windows OS.

As described in the [fast-glob v3.0.0 release notes](https://github.com/mrmlnc/fast-glob/releases?page=2), I think the reason is that only forward slashes are allowed in regular expressions in globs, and slash replacement is no longer processed.

So I added the process of replacing backslashes with forward slashes.